src/api/firebase.js

```diff
@@ -121,10 +122,11 @@ export async function updateItem(listId, item) {
 	});
 }
 
-export async function deleteItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to delete an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function deleteItem(listId, item) {
+	try {
+		await deleteDoc(doc(db, listId, item));
+	} catch (error) {
+		return false;
+	}
+	return true;
```

Comment:
Does this function always return `true`?

Reply:
i haven't been able to reproduce a way in which it would return false.

Reply:
What happens if you disable your internet?

Reply:
unfortunately the error doesn't pop up and once the internet is enabled again though, it deletes it and then pops up the confirmation correctly.  I'd like to go through maybe some testing scenarios tonight since we added the error handling back into it.